readthedocs/api/v3/views.py

```diff
@@ -304,3 +319,14 @@ def create(self, request, **kwargs):  # pylint: disable=arguments-differ
             data.update({'triggered': False})
             status = 400
         return Response(data=data, status=status)
+
+
+class BuildsCreateViewSet(BuildsViewSet):
```

Comment:
I don't quite follow this. What is this class for?

Reply:
This view is registered at URLs like `/api/v3/projects/pip/versions/v3.6.2/builds/` which allow "listing of all builds for that version" or "trigger a build for that version"

The view inherit `BuildsViewSet` which is registered for URLs like `/api/v3/projects/pip/builds/` that allow "listing of all builds for that project" or "trigger a build for the default version". The only different between these views is the way how they get the version to trigger a build.